apps/exchange/lib/exchange/matching_engine.ex

```diff
@@ -114,6 +114,22 @@ defmodule Exchange.MatchingEngine do
     GenServer.call(via_tuple(ticker), :total_ask_orders)
   end
 
+  @doc """
+  Returns the list of open orders
+  """
+  @spec open_orders(ticker) :: {atom, list()}
+  def open_orders(ticker) do
+    GenServer.call(via_tuple(ticker), :open_orders)
+  end
+
+  @doc """
+  Returns the list of open orders from a trader
+  """
+  @spec open_orders_by_trader(ticker) :: {atom, list()}
+  def open_orders_by_trader(ticker) do
```

Comment:
Here you need to receive as input the trader_id.
`@spec open_orders_by_trader(ticker, string.t()) :: {atom, list()}`
`def open_orders_by_trader(ticker, trader_id) do`

Reply:
Done